### DISCUSSION TITLE
How to reproduce/build JniLibs .so files

### DISCUSSION THREAD

**Discussion by ChristianGynnild** [lhns/restic-android#175] [Q&A]
There are 5 different .so files in the JniLibs folder:
libdata_libtalloc.so.2.so
libdata_loader.so
libdata_loader32.so (only in arm64-v8a and x86_64)
libdata_proot.so
libdata_restic.so

Which tools/commands were used to build the .so files?
Which repositories/source code was used and which version?
How to reproduce/build the .so files in the project for every build target? (arm64-v8a, armeabi-v7a, x86, x86_64)

**Comment by lhns** [comment]
fixed in 06e9efe138bb01da7923fe779b4dfb6a3668805f
